Source/WebKit/UIProcess/Network/NetworkProcessProxy.h

```diff
@@ -340,6 +340,7 @@ class NetworkProcessProxy final : public AuxiliaryProcessProxy, private ProcessT
     void didReceiveAuthenticationChallenge(PAL::SessionID, WebPageProxyIdentifier, const std::optional<WebCore::SecurityOriginData>&, WebCore::AuthenticationChallenge&&, bool, AuthenticationChallengeIdentifier);
     void negotiatedLegacyTLS(WebPageProxyIdentifier);
     void didNegotiateModernTLS(WebPageProxyIdentifier, const URL&);
+    void didLoadFailDueToNetworkConnectionIntegrity(WebPageProxyIdentifier, const URL&); 
```

Comment:
Nit - maybe either rename this to `loadDidFailDueToNetworkConnectionIntegrity`, or `didFailLoadDueToNetworkConnectionIntegrity`? ("didLoadFailDueToNetworkConnectionIntegrity" reads a bit awkwardly)